Sources/_CFXMLInterface/CFXMLInterface.c

```diff
@@ -1073,7 +1073,10 @@ CFStringRef _CFXMLNodeCopyPrefix(_CFXMLNodePtr node) {
     xmlChar* result = NULL;
     xmlChar* unused = xmlSplitQName2(_getQName((xmlNodePtr)node), &result);
 
-    CFStringRef resultString = __CFSwiftXMLParserBridgeCF.CFStringCreateWithCString(NULL, (const char*)result, kCFStringEncodingUTF8);
+    CFStringRef resultString = NULL;
+    if (result) {
+      __CFSwiftXMLParserBridgeCF.CFStringCreateWithCString(NULL, (const char*)result, kCFStringEncodingUTF8);
```

Comment:
Nit - inconsistent spacing in the file.

Is `xmlFree(NULL)` ok to call?

Reply:
Yes, AFAIK, `xmlFree` behaves as `free`, where `nullptr` as an argument is well defined.